nixos/modules/services/misc/portunus.nix

```diff
@@ -0,0 +1,280 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.portunus;
+
+in
+{
+  options.services.portunus = {
+    enable = mkEnableOption "Portunus, a self-contained user/group management and authentication service";
+
+    domain = mkOption {
+      type = types.str;
+      default = "";
```

Comment:
Why this `default` if the option must be set anyway? Remove this `default` and the associated assertion can go as well.